parking-lot/src/ParkingLot.ts

```
@@ -0,0 +1,69 @@
+    private activeTickets: Set<Ticket>;
+    private floors: ParkingFloor[];
+    private feeStrategy: FeeStrategy;
+
+    private constructor() {
+        this.floors = [];
+        this.activeTickets = new Set();
+        this.feeStrategy = new FlatFeeStrategy();
+    }
+
+    static getInstance() {
+        if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)